metagenomics_pipeline_qPCRnorm.ipynb

```diff
@@ -3649,7 +3649,9 @@
     "         'BarcodesAreRC': 'False',\n",
     "         'ForwardAdapter': 'GATCGGAAGAGCACACGTCTGAACTCCAGTCAC',\n",
     "         'ReverseAdapter': 'GATCGGAAGAGCGTCGTGTAGGGAAAGGAGTGT',\n",
-    "         'HumanFiltering': 'True'\n",
+    "         'HumanFiltering': 'True',\n",
+    "         'library_construction_protocol': '',\n",
```

Comment:
@RodolfoSalido for the **qPCRnorm** notebook what would be an appropriate value for `library_construction_protocol`. Would it be `Knight Lab Nextera prep` or `Knight Lab Kapa HP`?

Reply:
Knight Lab Nextera and Knight Lab KAPA HyperPlus would probably be best, for a more descriptive language. I don't think the suffix prep is needed.